.github/workflows/r-ci.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: R facebook survey
+
+on:
+  push:
+    branches: [ main, prod, 'release/*' ]
```

Comment:
I think we can remove the `'release/*' ` (also in the python one) since we don't use those anymore, is that right @krivard?

Reply:
That's correct